src/components/message/message-card/MessageCard.tsx

```diff
@@ -70,6 +71,7 @@ function MessageCardBase({ message, showRaw, showRawHandler }: Props) {
 		isSelected ? 'selected' : null,
 		isAttached ? 'attached' : null,
 		isPinned ? 'pinned' : null,
+		isHighlighted ? 'highlighted' : null,
```

Comment:
I thought we would have some effect that will reset hightlightedMessageId after applying highlight styles to it, because if we will not reset it might highlight again

Reply:
Fixed